source/guides/guides/web-security.md

```diff
@@ -264,5 +264,9 @@ Still here? That's cool, let's disable web security!
   "chromeWebSecurity": false
 }
 ```
+{% note danger %}
```

Comment:
Move this to the top of the '# Disabling Web Security' section so that it's more prominent